### ISSUE TITLE
Admin-UI: Provide a way to view and edit realm attributes

### ISSUE THREAD

**Issue by antikalk** [keycloak/keycloak#17732]
### Description

At the moment it is not possible to edit realm attributes (for example custom realm attributes that are used in extensions) in the admin ui realm settings.

In my opinion it would be nice to have a tab for attributes in the realm settings which lists all attributes of a realm and provides a way to edit/add or remove them. Similiar to user attributes e.g. 

### Motivation

Provide a non API way of editing realm attributes.

**Comment by sri-nayak** [comment]
yes we need configuration to add a custom configuraton tab in realm setting

**Comment by ozfox** [comment]
There is an example on how to implement this feature in the [keycloak-quickstarts](https://github.com/keycloak/keycloak-quickstarts/tree/latest/extension/extend-admin-console-spi) repository, in case anybody stumbles across this issue.
